homeassistant/components/leneda/__init__.py

```diff
@@ -0,0 +1,84 @@
+"""The Leneda integration."""
+
+from __future__ import annotations
+
+import logging
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import config_validation as cv
+from homeassistant.helpers.device_registry import DeviceEntry
+from homeassistant.helpers.typing import ConfigType
+
+from .const import CONF_ENERGY_ID, CONF_METERING_POINTS, DOMAIN
+from .coordinator import LenedaCoordinator
+
+_LOGGER = logging.getLogger(__name__)
+
+CONFIG_SCHEMA = cv.config_entry_only_config_schema(DOMAIN)
```

Comment:
I can't see this referenced in this file, whats it doing?

Reply:
Thank you, it was superfluous indeed. Removed.